vms/authentication/tests/test_services.py

```diff
@@ -1,3 +1,19 @@
 from django.test import TestCase
+from authentication.views import login_process
+from django.contrib.auth.models import User
+from django.test.client import RequestFactory
+
 
 # Create your tests here.
+class TestLoginProcess(TestCase):
+
+	def testLoginViewRateLimit(self):
+		request = RequestFactory().post("/login")
+		request.user = User.objects.create_user(
+			username='Test User', email='test@a.com', password='Password')
+		for loginAttemp in range(25):
```

Comment:
@smarshy @tapasweni-pathak 
I kept this 25 intentionally despite ratelimit being 50/minute.
For every login view access, there is a redirection which hit this view again.
so there are 2 login view access per iteration. so rate limit is hit 50 after 25 iteration.